person.rb

```diff
@@ -23,11 +23,6 @@ def correct_name
     name
   end
 
-  def add_rental(book, date)
-    rental = Rental.new(date, book, self)
-    @rentals.push(rental)
-  end
-
   private
```

Comment:
- [ ] I think there was a misunderstanding of what was to be deleted here. Don't worry. I'll tell you exactly what to add back to get approved. 
- Your project will not be complete if a  method for adding rentals is not present.
- This is how it should look like 

```
```def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

-  The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
- In your rental class, `date, book and person` is already initialized when you call the `.new` method. so no need to add the code `@rentals.push(rental)`